curation_concerns-models/app/jobs/audit_job.rb

```diff
@@ -41,12 +44,12 @@ def run_audit
 
       if fixity_ok
         passing = 1
-        ChecksumAuditLog.prune_history(id, path)
+        ChecksumAuditLog.prune_history(id, association_name)
       else
         logger.warn "***AUDIT*** Audit failed for #{uri} #{error_msg}"
         passing = 0
       end
-      ChecksumAuditLog.create!(pass: passing, generic_file_id: id, version: uri, dsid: path)
+      ChecksumAuditLog.create!(pass: passing, generic_file_id: id, version: uri, dsid: association_name)
```

Comment:
I also wonder why we don't just pass the ID of the File itself rather than the generic_file_id + association_name.  I don't think association_name is going to work well unless it's a singular association.